include/pybind11/detail/class.h

```diff
@@ -445,9 +445,15 @@ inline void clear_instance(PyObject *self) {
 /// Instance destructor function for all pybind11 types. It calls `type_info.dealloc`
 /// to destroy the C++ object itself, while the rest is Python bookkeeping.
 extern "C" inline void pybind11_object_dealloc(PyObject *self) {
+    auto *type = Py_TYPE(self);
```

Comment:
Do you actually need the `type` variable? It's only used once and `Py_TYPE(self)` could be inlined below.
If you prefer to keep: I think it's better to move it after the comment (and delete the new empty line).

Reply:
Oops, sorry, somehow I didn't see that you just moved the line. Good as is.